estate/models/estate_property.py

```diff
@@ -0,0 +1,43 @@
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
+    _description = "Real estate management"
+
+    active = True
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
```

Comment:
Always set the label of the field. even if it's obvious and even if the orm will generate one for you using the field name.
`postcode = fields.Char("Postcode") or postcode = fields.Char(string="Postcode")`
-> better the first in this case, unless string is not the first param. (typically for relational fields)